src/packages/console/src/pages/verifyEmailPage.tsx

```diff
@@ -46,32 +31,39 @@ export const VerifyEmailPage = (): React.ReactElement => {
   });
 
   return (
-    <Container component='main' maxWidth='sm'>
-      <Paper className={classes.paper}>
-        <Typography component='h1' variant='h5'>
+    <ContainingView isCenteredHorizontally>
+      <ResponsiveContainingView size={12} sizeResponsive={{ small: 8, medium: 6, large: 5 }}>
+        <Stack direction={Direction.Vertical} isFullHeight={true}>
+          <Stack.Item growthFactor={1} shrinkFactor={1} />
+          <Box variant='card' isFullWidth={false}>
+            <Text variant='header5' tag='h1' alignment={TextAlignment.Center}>
           Please check your email
-        </Typography>
-        <br />
-        <br />
-        <Typography>
+            </Text>
+            <br />
+            <br />
+            <Text>
           We need to verify your email before you can create and edit sites. It only takes a second, please check your inbox for an email from everypass@kibalabs.com.
-        </Typography>
-        <br />
-        <br />
-        {!verificationSent && (
-          <Button
-            variant='outlined'
-            onClick={onResendVerificationClicked}
-          >
-            Resend Verification
-          </Button>
-        )}
-        {verificationSent && (
-          <Typography color='textPrimary'>
+            </Text>
+            <br />
+            <br />
+            <Stack direction={Direction.Horizontal} contentAlignment={Alignment.Center}>
```

Comment:
why did you put this in a stack, i think it can just be removed here? if its for centering, use a stack item and put one around the button and one around the text, not one around both

Reply:
for reference, a `Stack` will create a new DOM element but a `Stack.Item` will not so the latter is preferable

Reply:
Actually they are rendered conditionally, so I had put the Stack.Item outside

Reply:
you've use a stack, not stack item. And what you should do is put the stack.item inside the condition, e.g. 
```             
 {!verificationSent && (
<Stack.Item alignment={Alignment.Center}>
  <Button
    isFullWidth={false}
    onClicked={onResendVerificationClicked}
    text='Resend Verification'
  />
<Stack.Item>
)}
```

Reply:
otherwise you are creating coupling between these two elements but they are not both centered together, each has its own reason for being centered. make sense?